main.rb

```diff
@@ -0,0 +1,175 @@
+# rubocop:disable Metrics
+require './book'
+require './classroom'
+require './rental'
+require './person'
+require './student'
+require './teacher'
+
```

Comment:
- [ ] Kindly ensure to add _appropriate space_ and _colon symbol_ between the statements and the values.

**Insuffient spacing and missing colon:**
![image](https://user-images.githubusercontent.com/12745474/151549989-66e31c11-d441-4441-acd4-82e022fca6f0.png)